src/cockpit/389-console/src/lib/replication/replAgmts.jsx

```diff
@@ -1193,31 +1195,122 @@ export class ReplAgmts extends React.Component {
         this.setState({
             savingAgmt: true,
         });
-        log_cmd('saveAgmt', 'update replication agreement', cmd);
+
+        const passwdFile = "/tmp/ds-passwd-file-" + new Date().getTime() + ".txt";
+        const bsPasswdFile = "/tmp/ds-bs-passwd-file-" + new Date().getTime() + ".txt";
+        const rm_cmd = ["rm", passwdFile, ];
+        const create_file_cmd = ["touch", passwdFile, bsPasswdFile];
+        const chmod_cmd = ["chmod", "600", passwdFile, bsPasswdFile ];
+        const update_cmd = [
+            '/bin/sh', '-c',
+            '/usr/bin/echo -e \'' + passwd + '\' >> ' + passwdFile
```

Comment:
I wonder if the command will still appear on `ps -ef` output...

I think we can use something from cockpit-spawn options:
https://cockpit-project.org/guide/latest/cockpit-spawn

Maybe something like this with .stream()?
https://github.com/389ds/389-ds-base/blob/main/src/cockpit/389-console/src/lib/monitor/replMonitor.jsx#L1116-L1177

Reply:
We can ask for the password with something like that also:
`read -s -p "Password: " password ; echo $password > passwdFile`

Or implement the "user input" functionality into our CLI.

Reply:
Yes, it's still visible:
```
2023/03/02 03:16:40 CMD: UID=0     PID=3438   | /bin/sh -c /usr/bin/echo -e 'test' >> /tmp/ds-passwd-file-1677745001625.txt
2023/03/02 03:16:40 CMD: UID=0     PID=3439   | /bin/sh -c /usr/bin/echo -e 'test' >> /tmp/ds-passwd-file-1677745001625.txt
```

Reply:
Ugh...  This is also how we've been calling dscreate in the UI for years.  Gonna have to make more changes to the CLI to support password prompting...